src/plugins/workspace/public/components/service_card/workspace_list_card.tsx

```diff
@@ -0,0 +1,192 @@
+      return orderBy(recentWorkspaces, ['timestamp'], ['desc'])
+        .filter((ws) => availableWorkspaces.some((a) => a.id === ws.id))
+        .slice(0, MAX_ITEM_IN_LIST)
+        .map((item) => ({
+          id: item.id,
+          name: availableWorkspaces.find((ws) => ws.id === item.id)?.name!,
+          time: item.timestamp,
+        }));
+    } else if (filter === 'updated') {
+      return orderBy(availableWorkspaces, ['lastUpdatedTime'], ['desc'])
+        .slice(0, MAX_ITEM_IN_LIST)
+        .map((ws) => ({
+          id: ws.id,
+          name: ws.name,
+          time: ws.lastUpdatedTime,
+        }));
+    }
+    return [];
+  }, [filter, availableWorkspaces]);
+
+  const handleSwitchWorkspace = (id: string) => {
+    const { application, http } = props.core;
+    if (application && http) {
+      navigateToWorkspaceDetail({ application, http }, id);
+    }
+  };
+
+  const { application } = props.core;
+
+  const isDashboardAdmin = application.capabilities.dashboards?.isDashboardAdmin;
+
+  return (
+    <EuiPanel paddingSize="s" hasBorder={false} hasShadow={false}>
+      <EuiFlexGroup gutterSize="xs" alignItems="center">
+        <EuiFlexItem grow={3}>
+          <EuiTitle>
+            <h4>Workspaces</h4>
+          </EuiTitle>
+        </EuiFlexItem>
+        <EuiFlexItem grow={1}>
+          <EuiToolTip position="bottom" content={WORKSPACE_LIST_CARD_DESCRIPTION}>
+            <EuiIcon type="iInCircle" aria-label="workspace list card description" />
+          </EuiToolTip>
+        </EuiFlexItem>
+        <EuiFlexItem grow={5}>
+          <EuiSelect
+            value={filter}
+            data-test-subj="workspace_filter"
+            onChange={(e) => {
+              setFilter(e.target.value);
+            }}
+            options={[
+              {
+                value: 'viewed',
+                text: i18n.translate('workspace.list.card.filter.viewed', {
+                  defaultMessage: 'Recently viewed',
+                }),
+              },
+              {
+                value: 'updated',
+                text: i18n.translate('workspace.list.card.filter.updated', {
+                  defaultMessage: 'Recently updated',
+                }),
+              },
+            ]}
+          />
+        </EuiFlexItem>
+        {isDashboardAdmin && (
+          <EuiFlexItem grow={false}>
+            <EuiToolTip position="top" content="Create workspace">
+              <EuiButtonIcon
+                data-test-subj="create_workspace"
+                aria-label="create workspace"
+                display="base"
+                iconType="plus"
+                size="m"
+                onClick={() => {
+                  application.navigateToApp(WORKSPACE_CREATE_APP_ID);
+                }}
+              />
+            </EuiToolTip>
+          </EuiFlexItem>
+        )}
+      </EuiFlexGroup>
+
+      <EuiSpacer />
+      <EuiListGroup flush={true} bordered={false} style={{ minHeight: '300px' }}>
```

Comment:
Nit: I'd suggest to have a future improvement task make this service card a flex container, so we don't need to specify a fix number 300px to stretch the list.